build.gradle

```diff
@@ -0,0 +1,34 @@
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.android.tools.build:gradle:1.3.0'
+  }
+}
+
+allprojects {
+  repositories {
+    mavenCentral()
+  }
+}
+
+ext {
+  minSdkVersion = 15
```

Comment:
Every 14 phone was updated to 15. There's no need to tiptoe around it. Stop supporting it.